python/nano/src/bigdl/nano/pytorch/trainer/Trainer.py

```diff
@@ -310,56 +311,58 @@ def trace(model: nn.Module, input_sample=None, accelerator=None):
-        :param input_sample: A set of inputs for trace, defaults to None if you have trace before or
-                             model is a LightningModule with any dataloader attached,
-                             defaults to None.
-                             Only ONNX and OpenVINO requires an input sample to export the model.
+        :param model: Any model of torch.nn.Module, including PytorchOpenVINOModel,
+         PytorchONNXModel.
+        :param path: Path to saved model. Path should be a directory.
         """
-        if accelerator == 'openvino':
-            if precision is None:
-                if not hasattr(model, "save") and isinstance(model, nn.Module):
-                    model = PytorchOpenVINOModel(model, input_sample=input_sample)
-                else:
-                    raise TypeError("Model type of {} can not be exported.".format(type(model)))
-                model.save(path)
+        os.makedirs(path, exist_ok=True)
+        if hasattr(model, 'save'):
+            model.save(path)
+        else:
+            with open(path + '/meta-data.yml', 'w+') as f:
+                metadata = {
+                    'ModelType': 'PytorchModel',
+                    'checkpoint': 'saved_weight.pt'
+                }
+                yaml.safe_dump(metadata, f)
+            torch.save(model.state_dict(), "{}/{}".format(path, metadata['checkpoint']))
 
     @staticmethod
-    def load(path, accelerator=None):
+    def load(path, model: LightningModule = None):
         """
         Load a model from local.
 
-        :param path: Path to model to be loaded. You need to specify path suffix carefully.
-                     For example, 'model.xml' for OpenVINO, 'model.onnx' for ONNX.
-        :param accelerator: Saved model format, defaults to None.
-                            accelerator=None will save Pytorch model, accelerator='openvino' will
-                            save an IR to inference by OpenVINO. Same as ONNX.
-        :return: A PytorchOpenVINOModel using openvino for inference if accelerator='openvino'
+        :param path: Path to model to be loaded. Path should be a directory.
+        :param model: Required FP32 model to load pytorch model. Optional for ONNX/OpenVINO.
+        :return: Model with different acceleration(None/OpenVINO/ONNX) or
+                 precision(FP32/FP16/BF16/INT8).
         """
-        if accelerator == 'openvino' or path.split('.')[-1] == 'xml':
-            # TODO: Need to fix this with lazy import class.
-            # The usage should be:
-            #   PytorchOpenVINOModel.load(path)
-            if not os.path.exists(path):
-                raise FileNotFoundError("{} doesn't exist.".format(path))
-            return PytorchOpenVINOModel(path)
+        if not os.path.exists(path):
+            raise FileNotFoundError("{} doesn't exist.".format(path))
+        with open(path + '/meta-data.yml', 'r') as f:
```

Comment:
Maybe using "os.path.join"

Reply:
What if '/meta-data.yml' does not exist?

Reply:
Raise error now.